compiler/rustc_builtin_macros/src/deriving/generic/mod.rs

```diff
@@ -1038,7 +1038,7 @@ impl<'a> MethodDef<'a> {
     /// `&self.x` because that might cause an unaligned ref. So for any trait
     /// method that takes a reference, we use a local block to force a copy.
     /// This requires that the field impl `Copy`.
-    /// ```
+    /// ```rust,ignore (example)
```

Comment:
Why ignoring this one?

Reply:
https://github.com/rust-lang/rust/pull/106621#discussion_r1085864118